examples/tv-casting-app/linux/main.cpp

```diff
@@ -135,6 +135,11 @@ int main(int argc, char * argv[])
         SetDeviceAttestationVerifier(GetDefaultDACVerifier(testingRootStore));
     }
 
+    // Enter commissioning mode, open commissioning window
+    static chip::CommonCaseDeviceServerInitParams initParams;
+    (void) initParams.InitializeStaticResourcesBeforeServerInit();
```

Comment:
why the void cast here? Why not VerifyOrDie like the method below that returns CHIP_NO_ERROR?

Reply:
this code is transplanted, and not clear what the history is for this. Please replace this with a VerifyOrDie in a future PR.

Reply:
Fixed in https://github.com/project-chip/connectedhomeip/pull/18701/files#diff-d3761c18745da28a2feffde261f8c90dace156bd67e6168a3181f8671de837e0R140